modules/ROOT/pages/sap/sap-s4-hana-connector-reference.adoc

```diff
@@ -40,15 +40,15 @@ Basic Authentication is the simplest way of enforcing access controls because it
 [cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
 |======================
 | Name | Type | Description | Default Value | Required
-| Service base URL a| String |  +++The base URL of the service on SAP S/4HANA to be accessed from the connector. From this base URL all other endpoints derive.+++ |  | *x*{nbsp}
-| Username a| String |  +++The account username with which to log into SAP S/4HANA.+++ |  | *x*{nbsp}
-| Password a| String |  +++The account password with which to log into SAP S/4HANA.+++ |  | *x*{nbsp}
+| Service base URL a| String |  +++The base URL of the service on SAP S/4 HANA to be accessed from the connector. From this base URL all other endpoints derive.+++ |  | *x*{nbsp}
+| Username a| String |  +++The account username with which to log into SAP S/4 HANA.+++ |  | *x*{nbsp}
+| Password a| String |  +++The account password with which to log into SAP S/4 HANA.+++ |  | *x*{nbsp}
 | Service discovery relative path a| String |  +++URL which will be used for automatic services discovery+++ |  +++sap/opu/odata/IWFND/CATALOGSERVICE;v=0002/ServiceCollection+++ | {nbsp}
 | Unlisted services a| Array of String |  +++List of services which will be added to the list of discovered services+++ |  | {nbsp}
 | Default headers a| Array of <<defaultHeader>> |  +++Default HTTP headers the message should include.+++ |  | {nbsp}
 | Default query parameters a| Array of <<defaultQueryParam>> |  +++Default Query parameters the request should include.+++ |  | {nbsp}
 | Proxy configuration a| <<ProxyConfiguration>> |  +++Proxy configuration for the connector.+++ |  | {nbsp}
-| TLS configuration a| <<Tls>> |  +++Protocol to use for communication. Valid values are HTTP and HTTPS. Default value is HTTP. When using HTTPS the HTTP communication is going to be secured using TLS / SSL. If HTTPS was configured as protocol then the user needs to configure at least the keystore in the tls:context child element of this listener-config.+++ |  | {nbsp}
+| TLS configuration a| <<Tls>> |  +++Protocol to use for communication. Valid values are HTTP and HTTPS. Default value is HTTP. When using HTTPS, the HTTP communication is going to be secured using TLS / SSL. If HTTPS was configured as protocol then the user needs to configure at least the keystore in the tls:context child element of this listener-config.+++ |  | {nbsp}
```

Comment:
Found one more - "...If HTTPS was configured as the protocol, the user needs to"
Please change if there are other occurrences of TLS configuration